### DISCUSSION TITLE
服务器配置问题

### DISCUSSION THREAD

**Discussion by yhy660** [accel-ppp/accel-ppp#202] [General]
下载accel-ppp后配置启动服务，
1.构建发现阶段PADI、PADR包，服务器正常回复PADO、PADS包，会话成功建立，
2.建立会话后服务器自动发送ppp lcp的configration request包，这个包不应该是客户端主动发送的吗，为什么服务器会主动请求？
3.这个configration request请求会发送十次，十次之后发送PADT包，会话结束，我自己构建的request包不支持认证，去除认证有回复但是服务器发送十次request包后，会话依然会终止，为什么？
4.在accel-ppp.conf里配置了认证选项，但无论是挑战握手还是密码认证服务器都会回configuration reject拒绝包，为什么？
5.在accel-ppp.conf里有pppoe模块的interface选项，为什么这个interface只能给lo（本地回环地址），只有设置这个接口服务器可以正常回包，给其他网卡接口服务器都不回包
![20240926-104459](https://github.com/user-attachments/assets/30ef7c55-aeeb-4fcd-8f1f-038fbcf4b1f2)

**Comment by svlobanov** [comment]
Hi @yhy660 
1. Please translate your request to English, auto-translated text is not clear
2. Please provide .pcap dump
3. Which client do you use?
4. Why do you have zero-mac ? Usually zero-mac causes issues on network level
5. where you dump ? on server? on client?
